independent-publisher-connectors/ORB Intelligence/apiDefinition.swagger.json

```diff
@@ -0,0 +1,4731 @@
+                      "confidence_score": 66,
+                      "match_mask": {
+                        "ein": "missing input",
+                        "main_name": "match",
+                        "corp_elem": "missing input",
+                        "other_names": "match",
+                        "address1": "missing input",
+                        "city": "missing input",
+                        "state": "missing input",
+                        "zip": "missing input",
+                        "country": "match",
+                        "webdomain": "missing input",
+                        "phone": "missing input"
+                      },
+                      "matched_fields": {
+                        "name": "Orb Intelligence",
+                        "country": "United States"
+                      }
+                    },
+                    {
+                      "result_number": 3,
+                      "orb_num": 5061292,
+                      "name": "ORB LLC",
+                      "entity_type": "company",
+                      "company_status": "active",
+                      "parent_orb_num": "null",
+                      "parent_name": "null",
+                      "ultimate_parent_orb_num": "null",
+                      "ultimate_parent_name": "null",
+                      "subsidiaries_count": 0,
+                      "branches_count": 0,
+                      "is_standalone_company": true,
+                      "address1": "288 BERRY ST",
+                      "city": "BROOKLYN",
+                      "state": "NY",
+                      "zip": "11211",
+                      "country": "United States",
+                      "iso_country_code": "US",
+                      "fetch_url": "http://api.orb-intelligence.com/3/fetch/5061292/?api_key=c66c5dad-395c-4ec6-afdf-7b78eb94166a",
+                      "confidence_score": 52,
+                      "match_mask": {
+                        "ein": "missing input",
+                        "main_name": "similar: 82%",
+                        "corp_elem": "missing input",
+                        "other_names": "similar: 82%",
+                        "address1": "missing input",
+                        "city": "missing input",
+                        "state": "missing input",
+                        "zip": "missing input",
+                        "country": "match",
+                        "webdomain": "missing input",
+                        "phone": "missing input"
+                      },
+                      "matched_fields": {
+                        "name": "ORB LLC",
+                        "country": "United States"
+                      }
+                    }
+                  ]
+                }
+              }
+            },
+            "description": "List of matched companies."
+          }
+        },
+        "tags": [
+          "Match and Fetch API"
+        ],
+        "summary": "Match",

```

Comment:
@aaryan2134 Can you please provide a sentence for all summary values, instead of just a word.